controllers/providers/metrics/prometheus.go

```diff
@@ -119,42 +174,86 @@ func (m *PrometheusMetrics) Unregister() {
 
 // init instantiates particular metrics
 func (m *PrometheusMetrics) init() {
-	m.metrics.HealthyRecords = prometheus.NewGaugeVec(
+	m.metrics.K8gbGslbHealthyRecords = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "healthy_records",
-			Help:      "Number of healthy records observed by K8GB.",
+			Name: K8gbGslbHealthyRecords,
+			Help: "Number of healthy records observed by K8GB.",
 		},
 		[]string{"namespace", "name"},
 	)
 
-	m.metrics.IngressHostsPerStatus = prometheus.NewGaugeVec(
+	m.metrics.K8gbGslbStatusPerIngressHosts = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "ingress_hosts_per_status",
-			Help:      "Number of managed hosts observed by K8GB.",
+			Name: K8gbGslbStatusPerIngressHosts,
+			Help: "Number of managed hosts observed by K8GB.",
 		},
 		[]string{"namespace", "name", "status"},
 	)
 
-	m.metrics.ZoneUpdateTotal = prometheus.NewCounter(
+	m.metrics.K8gbGslbErrorsTotal = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "delegated_zone_update",
-			Help:      "Number of delegated zone updates. Is hit for create as well",
+			Name: K8gbGslbErrorsTotal,
+			Help: "Number of errors",
 		},
+		[]string{"namespace", "name"},
 	)
 
-	m.metrics.ReconciliationTotal = prometheus.NewCounter(
+	m.metrics.K8gbGslbReconciliationLoopsTotal = prometheus.NewCounter(
 		prometheus.CounterOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "reconciliation_total",
-			Help:      "Number of successful reconciliation loops.",
+			Name: K8gbGslbReconciliationLoopsTotal,
+			Help: "Number of successful reconciliation loops.",
 		})
+
+	m.metrics.K8gbGslbFailoverStatus = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: K8gbGslbFailoverStatus,
+			Help: "K8GB Failover status.",
```

Comment:
Perhaps we can find a better description and maybe a name as well for this metric.
Because "K8GB Failover status." sounds a bit misleading to me, we're not talking about k8gb operator status here, also this is GaugeVec metric, not Scalar.
Maybe it should be "Gslb status count for Failover strategy"?
With "K8gbGslbStatusCountForFailover" and "k8gb_gslb_status_count_for_failover" respectively?

Same goes for Roundrobin and GeoIP metrics below

Reply:
@somaritane , amended, thx.

![Screenshot 2021-08-25 at 11 06 21](https://user-images.githubusercontent.com/7195836/130762283-ae1e7837-d88d-4d3e-8585-c71f49d9eae8.png)